tests/fixture/testnet/local/node.go

```diff
@@ -0,0 +1,322 @@
+// Copyright (C) 2019-2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package local
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"io/fs"
+	"net"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"syscall"
+	"time"
+
+	"github.com/spf13/cast"
+
+	"github.com/ava-labs/avalanchego/api/health"
+	cfg "github.com/ava-labs/avalanchego/config"
+	"github.com/ava-labs/avalanchego/ids"
+	"github.com/ava-labs/avalanchego/node"
+	"github.com/ava-labs/avalanchego/tests/fixture/testnet"
+	"github.com/ava-labs/avalanchego/utils/perms"
+)
+
+var errProcessNotRunning = errors.New("process not running")
+
+// Defines local-specific node configuration. Supports setting default
+// and node-specific values.
+//
+// TODO(marun) Support persisting this configuration per-node when
+// node restart is implemented. Currently it can be supplied for node
+// start but won't survive restart.
+type LocalConfig struct {
+	// Path to avalanchego binary
+	ExecPath string
+	// Whether a node is configured with static (e.g. 9560) or dynamic
+	// (e.g. 0) tcp ports for both staking and api connections.
+	UseStaticPorts bool
```

Comment:
is this being used here?

Reply:
It is a bit misleaving in the code, you read this and think node is using it. Each one should have different structs.